examples/serverless/pytorch/README.md

```diff
@@ -0,0 +1,15 @@
+<a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
+
+This is an end to end example of how to ingest training data, train a model and use it for predictions serverless-ly, by employing UDFs on TileDB-Cloud. 
+In this example, we work on the popular mnist dataset. Firstly, we ingest all training images and labels in TileDB arrays and register them 
```

Comment:
I get that it is implied but consider commenting that it is also related to a dense case.